packages/hebao_v1/contracts/stores/SecurityStore.sol

```diff
@@ -53,46 +55,93 @@ contract SecurityStore is DataStore
         view
         returns (bool)
     {
-        return wallets[wallet].guardianIdx[guardian] > 0;
+        return wallets[wallet].guardianIdx[guardian] > 0 &&
+            wallets[wallet].guardianEffectiveTime[guardian] >= now &&
+            wallets[wallet].guardianRemovalEffectiveTime[guardian] < now;
```

Comment:
If a guardian is added in `addOrUpdateGuardian` then `guardianRemovalEffectiveTime` is not set, so it will be 0. So either the logic here needs to be updated or `guardianRemovalEffectiveTime` needs to be set to some large value.

Depending on the implementation maybe `guardianValidFrom` and `guardianValidUntil` may be better names.

Reply:
w.guardianRemovalEffectiveTime[guardian] == 0 then it is < now, I can't see any problem here.
Yes, guardianValidFrom and guardianValidUntil are better names, I will update it.

Reply:
Oh right sorry. For some reason I always get these time related things wrong. :)